experimenter/experimenter/nimbus_ui_new/static/js/show_errors.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const urlParams = new URLSearchParams(window.location.search);
+  if (urlParams.get("show_errors")) {
+    const formIds = ["audience-form", "metrics-form"];
+
+    formIds.forEach(function (formId) {
+      const form = document.getElementById(formId);
+      if (form) {
+        form.requestSubmit();
```

Comment:
Oh I see, this is a way to force it into the validation state so it shows the errors?  Is there not a way we can check this in the template and add a CSS class that puts it into the validation state rather than doing it in JS this way?